prebuild.js

```diff
@@ -50,11 +50,11 @@ function prebuild (opts, target, runtime, callback) {
     ]
 
     if (opts.strip) {
-      tasks.splice(1, 0, function (filename, cb) {
-        buildLog('Stripping debug information from ' + filename)
-        strip(filename, function (err) {
+      tasks.splice(1, 0, function (filenames, cb) {
+        buildLog('Stripping debug information from ' + filenames)
```

Comment:
```suggestion
        buildLog('Stripping debug information from ' + filenames.join(', '))
```

Reply:
Thanks! Fixed in [81ffea1](https://github.com/prebuild/prebuild/pull/280/commits/81ffea1a70d60d6a818ca026cd001db84dd46291) from your suggestion 😄